web/profiles/joinup/joinup.info.yml

```diff
@@ -39,7 +40,6 @@ dependencies:
   - rdf_entity
   - rdf_export
   - rdf_taxonomy
-  - solution
   - state_machine
```

Comment:
What are you serious? That seems like a major bug to me! And a regression, because in Drupal 7 the dependencies were resolved during profile install if I remember correctly.

We need a followup for this, this needs to be investigated and reported in core, so that this can be solved. We cannot be tasked by manually ordering this list according to the dependency chain, that's not something a human should be dealing with, this is a machine's job!